examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md

```diff
@@ -1,101 +1,95 @@
+
+#### Prerequisites
+
+- Download and install the [Ballerina Distribution](https://ballerina.io/learn/getting-started/) relevant to your OS.
+- A Text Editor or an IDE
+  > **Tip**: For a better development experience, install one of the following Ballerina IDE plugins: [VSCode](https://marketplace.visualstudio.com/items?itemName=ballerina.ballerina), [IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina)
+- [cURL](https://curl.haxx.se) or any other REST client
+- Download the backend for Health Care System from [here](#).
+- If you did not try the [Transforming Message Content](../../transforming-message-content/transforming-message-content/) tutorial yet, you can clone the project from GitHub and follow the steps as mentioned below.
+
+#### Getting Started
+
+This tutorial includes the following sections.
+
+- [Scheduling an appointment](#scheduling-an-appointment)
+- [Adding payment for the appointment](#adding-payment-for-the-appointment)
+- [Sending response back to client](#sending-response-back-to-client)
+- [Testing the Implementation](#testing-the-implementation)
+
+#### Scheduling an appointment
 
-## Invoking the Service
+In the previous tutorial, we called the backend service from the add appointment resource itself. Since we are chaining two services in this example, we will add a util function to handle scheduling appointments at the backend.
 
-Let's start the service by navigating to the folder where the reserve_appointment_service.bal file is and issue the following command.
+<!-- INCLUDE_CODE_SEGMENT: { file: guide/health_care_service.bal, segment: segment_1 } -->
+
+When the client request is received, we check if the request payload is json. If so, we transform it to the format expected by the backend. Then we invoke the util function to add an appointment within the resource function.
+
+<!-- INCLUDE_CODE_SEGMENT: { file: guide/health_care_service.bal, segment: segment_2 } -->
+
+#### Adding payment for the appointment
+
+After invoking the appointment scheduling endpoint, we get the first response and check if it is an actual appointment confirmation response. If not, we simply throw an error. If it is a valid response, we call the function that invokes the payment settlement request.
+
+<!-- INCLUDE_CODE_SEGMENT: { file: guide/health_care_service.bal, segment: segment_3 } -->
+
+#### Sending response back to client
+
+Finally we get the response from payment endpoint and send it back to the user. We throw errors if the backend response or the original request payload are not in a valid format.
+
+#### Testing the Implementation
+
+Let's start the service by navigating to the folder _guide/health_care_service.bal_ file is and executing the following command.
```

Comment:
There is a problem in the site with this syntax "_guide/health_care_service.bal_"
<img width="975" alt="www-bug" src="https://user-images.githubusercontent.com/19742678/60684376-0fa42780-9ebb-11e9-986a-a22f7df20069.png">